cli/stream_command.go

```diff
@@ -1646,7 +1707,7 @@ func (c *streamCmd) showStreamConfig(cols *columns.Writer, cfg api.StreamConfig)
 	if cfg.MaxMsgSize == -1 {
 		cols.AddRow("Maximum Message Size", "unlimited")
 	} else {
-		cols.AddRow("Maximum Message Size", humanize.IBytes(uint64(cfg.MaxMsgSize)))
```

Comment:
missed one

Reply:
Done

Reply:
think this one still needs its humanize back

Reply:
It's getting late and it's been a long day 😅, actually done.